cluster-service/deploy/integration/cluster-service-namespace.yaml

```diff
@@ -4,13 +4,19 @@ kind: Template
 metadata:
   name: cluster-service-admin
 parameters:
-- name: NAMESPACE
-  description: The namespace to create
-  required: true
-  value: cluster-service-admin
-- name: CLIENT_ID
-  description: The Azure Client ID used for federation
-  required: true
+  - name: NAMESPACE
+    description: The namespace to create
+    required: true
+    value: cluster-service-admin
+  - name: CLIENT_ID
+    description: The Azure Client ID used for federation
+    required: true
+  - name: ORPHANED_NAMESPACE_CLEANER
```

Comment:
The component name is ORPHANED_NAMESPACE_CLEANER. Therefore, to designate the namespace of that component I would expect the parameter to be something like NAMESPACE_ORPHANED_NAMESPACE_CLEANER or ORPHANED_NAMESPACE_CLEANER_NAMESPACE

Reply:
I have addressed in the new PR: 
https://github.com/Azure/ARO-HCP/pull/1412/files
renamed it to `ORPHANED_NAMESPACE_CLEANER_NAMESPACE`